mapping.csv

```diff
@@ -403,6 +417,7 @@
 "^(ircbot|clicker|(ssl-)?unknown-bot-test|downloader|iframe exploit)$",malware-generic,shadowserver
 "^(jadtre|killav)$",jadtre,https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=TrojanDownloader:Win32/Jadtre.A
 "^renos$",renos,https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=Win32%2FRenos
+"^daws$",daws,https://www.microsoft.com/en-us/wdsi/threats/threat-search?query=daws
```

Comment:
malpedia maps it to sanny: https://malpedia.caad.fkie.fraunhofer.de/details/win.sanny

Reply:
Daws is a generic detection that may have overlapped with sanny. But it's more generic than that

Reply:
yeah, I think at the time I based that association on this blog post: http://contagiodump.blogspot.com/2012/12/end-of-year-presents-continue.html - I will remove it then! thx for the pointer!